HighAvailabilityModule.Storage.Client/SQLStorageMembershipClient.cs

```diff
@@ -88,6 +88,48 @@ public async Task <(string value, string type)> GetDataEntryAsync(string path, s
             }
         }
 
+        public object TryParseDataEntry(string value, string type)
+        {
+            if (type == "System.Guid")
+            {
+                return Guid.Parse(value);
+            }
+            else if (type == "System.String")
+            {
+                return value;
+            }
+            else if (type == "System.Int32")
+            {
+                return Int32.Parse(value);
+            }
+            else if (type == "System.Int64")
+            {
+                return Int64.Parse(value);
+            }
+            else if (type == "System.Double")
+            {
+                return Double.Parse(value);
+            }
+            else if (type == "System.String[]")
+            {
+                return value.Split(",".ToCharArray());
+            }
+            else if (type == "System.Byte[]")
+            {
+                string[] s = value.Split(",".ToCharArray());
+                byte[] valueByte = new byte[s.Length];
+                for (int i = 0; i < s.Length; i++)
+                {
+                    valueByte[i] = byte.Parse(s[i]);
+                }
+                return valueByte;
```

Comment:
use [existing API](https://stackoverflow.com/questions/1003275/how-to-convert-utf-8-byte-to-string) for this